### PR TITLE
Fizzbuzz tree

### PR DESCRIPTION
**Pull request by herndont**
- [x] Write a function called `FizzBuzzTree` which takes a tree as an argument.
- [x] Without utilizing any of the built-in methods available to your language, determine weather or not the value of each node is divisible by 3, 5 or both, and change the value of each of the nodes:
- [x] If the value is divisible by 3, replace the value with “Fizz”
- [x] If the value is divisible by 5, replace the value with “Buzz”
- [x] If the value is divisible by 3 and 5, replace the value with “FizzBuzz”
- [x] create lower level readme and update higher level readme
- [ ] For each method that you define, write test assertions for the following conditions at minimum:
          -“Happy Path” - Expected outcome
          - Expected failure
          - Edge Case (if applicable/obvious)

### CODE REVIEW

File: src/main/java/FizzBuzzTree/FizzBuzzTree.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        while (queue.isEmpty() == false) {
+            Node poppedNode = queue.remove();
+            for (int i = 0; i < poppedNode.nodes.size(); i++) {
+                queue.add(poppedNode.nodes.get(i));
+            }
+            if (((int)poppedNode.data % 3) == 0) {
+                poppedNode.data = ("Fizz");
+            } if (((int)poppedNode.data % 5) == 0) {
+                poppedNode.data = ("Buzz");
+            } if (((int)poppedNode.data % 3 & (int)poppedNode.data % 5) == 0){
```
Comment (JessLovell) [inline src/main/java/FizzBuzzTree/FizzBuzzTree.java:23]: Syntax error: &&

File: src/main/java/FizzBuzzTree/FizzBuzzTree.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package FizzBuzzTree;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class FizzBuzzTree {
```
Comment (JessLovell) [inline src/main/java/FizzBuzzTree/FizzBuzzTree.java:7]: No tests for this class
